styles/styles.css

```
@@ -399,17 +623,35 @@ body {
   align-items: center;
 }
 
+
 .social-item a {
-  color: #000000;
   text-decoration: none;
-}
-
-.social-item {
   width: 47px;
   height: 40px;
-  display: flex;
-  justify-content: center;
-  align-items: center;
+  display: block;
+}
+
+.social-item svg {
+  padding: 13px 12px;
+}
+
+.social-item svg {
+  color: #83b3d3;
+}
+
+.social-item a :hover {
+  color: #68a2ca;
+}
+
+/* не работает focus */
+
+.social-item a :focus {
```

Потому что ошибка в записи:
```suggestion
.social-item a:focus {
```

---

поставил специально зеленый цвет на состояние , не отображает. Пробел убрал не заработало.